gen-tc.sh

```diff
@@ -92,6 +92,7 @@ Supported platforms:
 	kobov5
 	nickel
 	remarkable
+  remarkable-aarch64
```

Comment:
Looks like a tabs vs spaces thing. ;-)

Reply:
Yep, just found out my neovim config does that so will have to fix that somewhere else